conformance/README.md

```diff
@@ -607,33 +607,51 @@ expectation, ending that branch of the test tree.
 These rules describe Ion data-model results for use in the `denotes` expectation:
 
 ```ebnf
-model-value   ::=  model-content  |  annotated
-
-model-content ::=  null.null
-                |  bool
-                |  int
-                |  string
-                |  "("  "null"    model-type    ")"
-                |  "("  "string"  codepoint*    ")"
-                |  "("  "symbol"  model-symtok  ")"
-                |  "("  "list"    model-value*  ")"
-                |  "("  "sexp"    model-value*  ")"
-                |  "("  "struct"  model-field*  ")"
-                |  "("  "blob"    bytes*        ")"
-                |  "("  "clob"    bytes*        ")"
+model-value     ::=  model-content  |  annotated
+
+model-content   ::=  null.null
+                  |  bool
+                  |  int
+                  |  string
+                  |  "("  "null"      model-type       ")"
+                  |  "("  "bool"      bool             ")"
+                  |  "("  "int"       int              ")"
+                  |  "("  "float"     string           ")" // See https://amazon-ion.github.io/ion-docs/docs/float.html
+                  |  "("  "decimal"   model-decimal    ")"
+                  |  "("  "timestamp" model-timestamp  ")"
+                  |  "("  "string"    codepoint*       ")"
+                  |  "("  "symbol"    model-symtok     ")"
+                  |  "("  "list"      model-value*     ")"
+                  |  "("  "sexp"      model-value*     ")"
+                  |  "("  "struct"    model-field*     ")"
+                  |  "("  "blob"      bytes*           ")"
+                  |  "("  "clob"      bytes*           ")"
 
 // TODO Other types per denotational semantics
 
-codepoint     ::=  int                               // in the range 0..0x10FFFF
+codepoint       ::=  int                                   // in the range 0..0x10FFFF
 
-model-symtok  ::=  string
-                |  int
-                |  "("  "text"  codepoint*  ")"
-                |  "("  "absent"  string  int  ")"       // symtab name + offset
+model-symtok    ::=  string
+                  |  int
+                  |  "("  "text"  codepoint*  ")"
+                  |  "("  "absent"  string  int  ")"       // symtab name + offset
 
-model-field   ::=  "("  model-symtok  model-value  ")"
+model-field     ::=  "("  model-symtok  model-value  ")"
 
-annotated     ::=  "("  "annot"  model-content  string*  ")"
+model-decimal   ::= int int                                // coefficient + exponent
+                  | "negative_zero" int
+
+model-timestamp ::= int
+                  | int int
+                  | int int int
+                  | int int int int int offset?
+                  | int int int int int int offset?
+                  | int int int int int decimal offset?
+
+offset          ::= "Z"
+                  | string                                 // content matches ^[+-]\d{2}:\d{2}$
```

Comment:
I think this should be handled numerically, so it doesn't rely on either parsing this string, or generating it, from the numeric forms.

That also means we need a way to denote the unknown offset.

Reply:
How about `(offset int int)` or `(offset_minutes int)`? Unknown offset can just be the absence of any offset clause in the timestamp clause.